src/cephadm/cephadm

```diff
@@ -2306,6 +2306,13 @@ def command_bootstrap():
                 logger.info('Deploying %s service with default placement...' % t)
                 cli(['orch', 'apply', t])
 
+        if not args.skip_monitoring_stack:
+            logger.info('Enabling mgr prometheus module...')
+            cli(['mgr', 'module', 'enable', 'prometheus'])
+            for t in ['prometheus', 'grafana', 'node-exporter', 'alertmanager']:
+                logger.info('Deploying %s service with default placement...' % t)
+                cli(['orch', 'apply', t])
```

Comment:
Hm. Do we really want to co-locate MON+MGR and the monitoring stack on the same host by default?

cc @l-mb ?

Reply:
IMO these are small daemons.  And pretty trivial to relocate later by adjusting the placement spec.